lottie/src/main/java/com/airbnb/lottie/animation/content/ContentGroup.java

```diff
@@ -148,25 +166,37 @@ Matrix getTransformationMatrix() {
     return path;
   }
 
-  @Override public void draw(Canvas canvas, Matrix parentMatrix, int parentAlpha) {
+  @Override public void draw(Canvas canvas, Matrix parentMatrix, int parentAlpha, boolean isOffScreenRenderingEnabled) {
```

Comment:
Pretty much all of these classes have access to LottieDrawable. I think you could just check it for `isOffScreenRenderingEnabled` instead of passing it around everywhere

Reply:
Exactly! I'll address it :)